app/sanitizers/rendered_markdown_scrubber.rb

```diff
@@ -30,12 +30,19 @@ def scrub_attributes(node)
 
       scrub_css_attribute(node)
     else
+      scrub_valid_attributes(node)
       super
     end
   end
 
   private
 
+  def scrub_valid_attributes(node)
+    node.attributes.each_value do |attribute|
+      attribute.value = attribute.value.gsub(/\{%|%\}/, "")
```

Comment:
Wow, I hate to be dense - but what does this regexp do? (largely my fault, I avoid regexps when possible and then act surprised when I see them, rather than there being something inherently confusing about your choice.)

Assuming it takes more than 5-10 seconds of thought for someone comfortable with regexes, does this warrant an explanatory comment?

Reply:
Actually, rather than a comment - what if this regex had an intention revealing name (as a class constant) that acted as good as a comment would? Reading ahead to the spec I see what the goal was - would 

`LIQUID_TAG_SYNTAX_REGEX` be a crazy name for this?